features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/components/RoomSummaryRow.kt

```diff
@@ -232,13 +231,12 @@ private fun NameAndTimestampRow(
 private fun InviteSubtitle(
     isDm: Boolean,
     inviteSender: InviteSender?,
-    canonicalAlias: RoomAlias?,
     modifier: Modifier = Modifier
 ) {
     val subtitle = if (isDm) {
         inviteSender?.userId?.value
```

Comment:
Could be `inviteSender?.userId?.value?.takeIf { isDm }`, maybe.

Reply:
Yes, I was expecting this remark :)